.github/workflows/continuous-integration.yml

```diff
@@ -122,8 +120,6 @@ jobs:
           name: ${{ env.ARTIFACTS_KADAI_WEB_NAME }}
           path: ${{ env.ARTIFACTS_KADAI_WEB_PATH }}
           if-no-files-found: error
-      - name: Remove kadai artifacts from cache
-        run: rm -rf ~/.m2/repository/io/kadai
```

Comment:
Same here. I think the removal makes sense. But you could replace the hardcoded path with `${{ env.ARTIFACTS_KADAI_JARS_PATH }}` to be consistent with the `compile_backend` job.